pandas/core/arrays/categorical.py

```diff
@@ -2001,15 +2001,12 @@ def __getitem__(self, key):
             else:
                 return self.categories[i]
 
-        if is_list_like(key) and not is_array_like(key):
-            key = np.asarray(key)
-
-        if com.is_bool_indexer(key):
-            key = check_bool_array_indexer(self, key)
+        if is_list_like(key) and not isinstance(key, tuple):
```

Comment:
i would handle the is_list_like case *inside* the check_array_indexer (I think you said this elsewhere as well)

Reply:
elsewhere we do `type(key) is not tuple` (I think to handle NamedTuple different), does that matter here?

Reply:
> elsewhere we do type(key) is not tuple (I think to handle NamedTuple different), does that matter here?

I think the tuple vs named tuple does not matter here, as here it's about numpy / positional indexing. The NamedTuple issue came up in label-based indexing I think, where a NamedTuple will be interpreted as a label (I assume?)